README.md

```diff
@@ -6,10 +6,10 @@ Symbolic (Java) PathFinder:
 This JPF extension provides symbolic execution for Java bytecode.
 It performs a non-standard interpretation of byte-codes.
 It allows symbolic execution on methods with arguments of basic types
-(int, long, double, boolean, etc.). It also supports symbolic strings, arrays, 
+(int, long, double, boolean, etc.). It also supports symbolic strings, arrays,
 and user-defined data structures.
 
-SPF now has a "symcrete" mode that executes paths 
+SPF now has a "symcrete" mode that executes paths
 triggered by concrete inputs and collects constraints along the paths
 
 A paper describing Symbolic PathFinder appeared at ISSTA'08:
```

Comment:
Can we change the format of this reference?

**Combining Unit-level Symbolic Execution and System-level Concrete Execution for Testing NASA Software**
_Corina S. Pǎsǎreanu, Peter C. Mehlitz, David H. Bushnell, Karen Gundy-Burlet, Michael Lowry, Suzette Person, and Mark Pape._  In Proceedings of the 2008 International Symposium on Software Testing and Analysis (ISSTA '08). https://doi.org/10.1145/1390630.1390635

```
@inproceedings{spf,
   author = {P{\u{a}}s{\u{a}}reanu, Corina S. and Mehlitz, Peter C. and Bushnell, David H. and Gundy-Burlet, Karen and Lowry, Michael and Person, Suzette and Pape, Mark},
   title = {Combining Unit-Level Symbolic Execution and System-Level Concrete Execution for Testing Nasa Software},
   year = {2008},
   isbn = {9781605580500},
   publisher = {Association for Computing Machinery},
   address = {New York, NY, USA},
   url = {https://doi.org/10.1145/1390630.1390635},
   doi = {10.1145/1390630.1390635},
   booktitle = {Proceedings of the 2008 International Symposium on Software Testing and Analysis},
   pages = {15–26},
   numpages = {12},
   keywords = {software model checking, symbolic execution, unit testing, system testing},
   location = {Seattle, WA, USA},
   series = {ISSTA '08}
}
```